README.md

```diff
@@ -142,75 +140,58 @@ _**NOTE:** Published versions can be installed from the [Chrome Web Store](
 
 ## Publishing to the Chrome Web Store
 
-1. Prepare Code
-   1. Ensure that all of the code, that should be published, has been merged into the master branch.
-   2. Ensure that the code in master is working as expected
+1. Prepare Code.
+   1. Ensure that all of the code that should be published has been merged into the master branch.
+   2. Ensure that the code in the master is working as expected.
```

Comment:
I think the commas in 1.1 help the sentence read more clearly; describing which code.  In. 1.2. you could say "the master branch" but we tend to just refer to this as "master"

Reply:
I will change it.